src/edu/csus/ecs/pc2/core/report/ContestCompareModel.java

```diff
@@ -0,0 +1,821 @@
+     * @param eventFeedJSON
+     * @throws IOException
+     * @throws JsonMappingException
+     * @throws JsonParseException
+     */
+    private void createComparison(String ndjson) throws JsonParseException, JsonMappingException, IOException {
+        String[] lines = ndjson.split("\\n");
+        createComparison(lines);
+
+    }
+
+    /**
+     * Load event feed lines into local classes
+     * 
+     * @param lines
+     * @throws IOException
+     * @throws JsonMappingException
+     * @throws JsonParseException
+     */
+    protected void createComparison(String[] lines) throws JsonParseException, JsonMappingException, IOException {
+
+        
+//        String filename = "stuf-event-feed."+new Date().getTime()+".json";
+//        FileUtilities.writeFileContents(filename, lines);
+//        System.out.println("write event feed to "+filename);
+        
+        int eventCount = 0;
+        
+        for (String event : lines) {
+            
+            eventCount ++;
+
+            if (event.length() == 0) {
+                // EOF
+                break;
+            }
+
+            Map<String, Object> eventMap = JSONUtilities.getMap(event);
+            ObjectMapper mapper = getMapper();
+
+            if (eventMap == null) {
+                // could not parse event
+                getLog().log(Level.WARNING, "Could not parse event: " + event);
+                if (Utilities.isDebugMode()) {
+                    System.out.println("Could not parse event: " + event);
+                }
+
+            } else {
+
+                CLICSEventFeedEvent eventFeedEntry = (CLICSEventFeedEvent) mapper.readValue(event, CLICSEventFeedEvent.class);
+                // find event type
+                String eventType = eventFeedEntry.getType();
+
+//                if (isAfterConfig(eventType)) {
+//                    break;
+//                }
+
+//                System.out.println(" event "+eventType+" "+eventFeedEntry.getData());
+
+                if (eventFeedEntry.getData() == null) {
+                    
+                    // SOMEDAY handle delete or whatver op has null data
+                    // log for now.
+                    getLog().log(Level.FINE, "Line: "+eventCount+ " data:null in event line '"+event+"'");
+                    
+                }
+                else if (CLICSEventType.LANGUAGES.toString().equals(eventType)) {
+                    // languages
+
+                    // languages data = {name=C, id=c}
+
+                    CLICSLanguage language = mapper.convertValue(eventFeedEntry.getData(), CLICSLanguage.class);
+
+                    feedLanguages.add(language);
+
+                } else if (CLICSEventType.JUDGEMENT_TYPES.toString().equals(eventType)) {
+                    // judgement-types
+
+                    // {"type": "judgement-types", "id": "a17b7d9d-43fc-43ed-bfd1-ef7fccbc6feb", "op": "create", "data": {"id": "RTE", "name": "Run Time Error", "solved": false, "penalty": true}}
```

Comment:
Use 2022-07 example in the comment, not the old feed.

Reply:
Noted.  No change at this time.